Remove PoC and revamp Samples (#412)
open-telemetry/opentelemetry-dotnet-instrumentation: docs/developing.md

./samples/ConsoleApp/bin/Debug/netcoreapp3.1/ConsoleApp
```

### Run Samples

> The sample applications can be used to see the local changes in action, see the

Reviewer: ```suggestion
You can use the sample applications to see the local changes in action. See the
```
